components/status/StatusActionsMore.vue

```diff
@@ -32,6 +33,11 @@ const userSettings = useUserSettings()
 const useStarFavoriteIcon = usePreferences('useStarFavoriteIcon')
 
 const isAuthor = $computed(() => status.account.id === currentUser.value?.account.id)
+const statusRoute = $computed(() => {
+  const route = getStatusRoute(status)
+  return withHttps(route.href.slice(1))
+})
```

Comment:
This is how the link is generated when you click on the item.  It is a relative link `/something` so I am just slicing off the first `/` and generating an https link.